tasks/main.yml

```diff
@@ -2,19 +2,19 @@
 - name: Install local facts
   ansible.builtin.include_tasks: facts.yml
 
-- name: Configure systemd resolved
-  ansible.builtin.include_tasks: resolvedconf.yml
+# - name: Configure systemd resolved
+#   ansible.builtin.include_tasks: resolvedconf.yml
```

Comment:
The reason for each set of commented-out tasks should be explained via a comment.

Reply:
I would ask for Dev to assist with this, then - as I am following this:
https://github.com/cisagov/ansible-role-hardening/blob/master/tasks/main.yml

Reply:
@bra1ncramp I think you can just put something generic for each commented-out section like:
```
# These tasks are not currently needed for our use case
```